build.gradle

```diff
@@ -100,6 +100,10 @@ rat {
     "**/*.json",
 
     // Katas files
+    "learning/katas/**/course-remote-info.yaml",
```

Comment:
Why are these excluded? All source files of the project should be included. Only autogenerated files should be excluded and I can see this captures some source files.